client/src/chess/startup/index.js

```diff
@@ -1,49 +1,23 @@
 var angular = require('angular');
 var ngRoute = require('angular-route');
-var AuthComponent = require('../auth/auth.component');
+var startupComponent = require('./startup.component')
+var ui = require('angular-ui-bootstrap');
+var ngAimate = require('angular-animate');
 
 module.exports = angular.module('chess.startup', [
-  ngRoute
+  ngRoute,
+  ui,
+  ngAimate
```

Comment:
just replace with `require('angular-animate')` because you really don't use variable `ngAnimate` anywhere.